docs/index.rst

```diff
@@ -28,6 +26,30 @@ Useful links
 
 * `Gitter Chat <https://gitter.im/ethereum/solidity/>`_
 
+Available Solidity Integrations
+-------------
+* `Browser-Based Compiler <https://chriseth.github.io/browser-solidity/>`_
+	WEB compiler for solidity language which could also simulate function calls.	
+
+* `Ethereum Studio <https://live.ether.camp/>`_
+	A fully-functional well-designed WEB IDE for Ethereum Solidity developers.
```

Comment:
`Specialized web IDE that also provides shell access to a complete Ethereum environment.`